src/controllers/top.js

```diff
@@ -19,7 +19,7 @@ topController.get = async function (req, res, next) {
 		data.title = `[[pages:top-${term}]]`;
 	}
 
-	const feedQs = data.rssFeedUrl.split('?')[1];
+	const [feedQs] = data.rssFeedUrl.split('?');
```

Comment:
Should be `const [, feedQs] = data.rssFeedUrl.split('?');` ?

Reply:
`eslint prefer-destructuring` Im not sold on, can lead to bugs, I think previous version is more readable

Reply:
I can disable it for arrays